src/Runner.php

```diff
@@ -229,12 +229,6 @@ public function runPHPCBF()
      */
     public function checkRequirements()
     {
-        // Check the PHP version.
-        if (PHP_VERSION_ID < 50400) {
-            $error = 'ERROR: PHP_CodeSniffer requires PHP version 5.4.0 or greater.'.PHP_EOL;
```

Comment:
This one looks like it should not be removed. This is the error which will be thrown if for whatever reason someone run PHPCS 3.x on PHP 5.3.

Reply:
The requirement is set in composer and I think it should be enough.

Reply:
@webimpress By far not all people use PHPCS via Composer, so I don't necessarily agree.

Reply:
This check needs to stay.

Reply:
Ok then, should I revert it or you will do it on merge?

Reply:
I've just noticed the autoloader fails for < 5.4, so leave it. I think I need to move the requirement check higher up.